resources/tests.mjs

```diff
@@ -894,6 +894,19 @@ Suites.push({
     ],
 });
 
+Suites.push({
+    name: "NewsSite-PostMessage",
+    url: "resources/newssite/news-next/dist/index.html",
+    tags: ["experimental", "newssite", "language"],
+    disabled: true,
+    async prepare() {},
+    type: "remote",
+    config: {
+        // name: "default", // optional param to target non-default tests locally
+        steps: 3, // manual step for the progress display on the bottom: this is the number of BenchmarkSteps in a BenchmarkSuite.
```

Comment:
Hm... so whenever we update benchmark suite, we need to remember to update this magic number?
That doesn't sound like a scalable solution.

Can we load `resources/newssite/news-next/src/workload-test.mjs` in the harness as well and get the number of suites from there?

Alternatively, can we define some kind of JSON file somewhere which outlines each step of suite?

Reply:
An alternative solution we discussed is a little shared helper library that communicates back the progress / step sizes on Test setup.
Given that we don't really have many workloads yet I'd prefer sticking to slightly more cumbersome manual approach and work towards a more maintainable solution in the long run.

Reply:
+1 on doing this as a followup. We can discuss there whether JSON manifest with hardcoded step count or encoding it in the setup message would be preferable.

Reply:
removed steps magic number!